Assets/TextAssets/TXT/AMemory.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;

```

Comment:
Can you include a class header w/ your name and a brief summary of the class. I see you began it below. Ideally, should be written at the very top of the file